docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -87,3 +87,5 @@ Taking this action completely eliminates the node, as follows:
 
 1. Removes the node record from the universe metadata.
 2. Updates metadata in the database only.
+
+You can recover a deleted node by following instructions provided in [Use Yugabyte Platform to add a node](../add-nodes/).
```

Comment:
Reading this page, not clear to me that you can recover a 'deleted' node; in Add Node it says you can add back a node in 'decommissioned' status, which comes after Release Node Instance steps. Deleted node may be gone for good? Needs clarification.

Reply:
Good catch. I moved that sentence to under "Release node instance".
(this whole article was really carefully crafted based on the input from dev, as internal node states and displayed statuses weren't always a match)